ci-operator/step-registry/openshift/e2e/metal/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- staebler
+- patrickdillon
+- jhixson74
+- jstuever
+- rna-afk
```

Comment:
For all of these owners files can we remove @staebler, and add @kirankt @sadasu & @barbacbd